src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/DataArchiveValidationServiceTests.cs

```diff
@@ -517,7 +380,65 @@ public async Task ValidateBulkDataArchiveFile_Fail_UnusedFilesInZip()
                     ]);
             }
 
-            VerifyAllMocks(fileUploadsValidatorService, fileTypeService);
+            VerifyAllMocks(fileUploadsValidatorService);
+        }
+
+        [Fact]
+        public async Task IsValidZipFile_NoFileProvided_ReturnsValidationError()
+        {
+            // Arrange
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId);
+
+            var service = SetupDataArchiveValidationService(contentDbContext: contentDbContext);
+
+            IFormFile? archive = null;
+
+            // Act
+            var result = await service.IsValidZipFile(archive!);
+
+            // Assert
+            Assert.Single(result);
+            Assert.Equal("FileIsNull", result[0].Code);
+            Assert.Equal("Argument provided for parameter 'zipFile' is null.", result[0].Message);
```

Comment:
This needs to be updated to match the new error message

Could also utilise `AssertHasErrors` here like I suggest in another comment